demo/src/main.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 import ReactDOM from 'react-dom/client'
 
-import { App } from './app.tsx'
+import { App } from './App';
```

Comment:
Let's also keep the naming conventions as is. Renaming the components won't add too much value and it also violates the current conventions in the repo